### PR TITLE
Exporter Install `readme` + `codelab` [AARD-1742]

### PR DESCRIPTION
**Pull request by BrandonPacewic**
Intermediate step towards #1073 and [AARD-1742](https://jira.autodesk.com/browse/AARD-1742). Steps to manually install the exporter while I finish working on the actual installers. Additionally all the old installer code was removed.

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1742)

### CODE REVIEW

File: installer/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Installing the Synthesis Fusion Exporter
+
+## Manual Install
+
+- Navigate to [`synthesis.autodesk.com/download`](https://synthesis.autodesk.com/download.html).
+- Find the Exporter source code zip download.
+  - Note that the source code is platform agnostic, it will work for **both** `Windows` and `Mac`.
+- Once the source code for the Exporter is installed unzip the folder.
```
Comment (azaleacolburn) [inline installer/README.md]:
```suggestion
- Once the source code for the Exporter is downloaded, unzip the folder.
```

File: installer/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Next, if not done already, install `Autodesk Fusion`.
```
Comment (azaleacolburn) [inline installer/README.md]:
```suggestion
- Next, if you haven't already, install `Autodesk Fusion`.
```

File: installer/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Once Fusion is open navigate to the `Utilities Toolbar`.
```
Comment (azaleacolburn) [inline installer/README.md]:
```suggestion
- Once Fusion is open, navigate to the `Utilities Toolbar`.
```

File: installer/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+![image_caption](../tutorials/img/fusion/fusion-empty.png)
+- Click on `Scripts and Add-ins` from the toolbar.
```
Comment (azaleacolburn) [inline installer/README.md]:
```suggestion
- Click on `Scripts and Add-ins` in the toolbar.
```

File: tutorials/FusionExporter.md [hunk 5]
```diff
@@ -15,11 +15,40 @@ The Synthesis Fusion 360 exporter is the tool used by both developers and users
...
 For information regarding the manual install process visit the [Synthesis Fusion 360 Exporter](https://github.com/Autodesk/synthesis/tree/prod/exporter) source code for more information.
 
-### Getting Started
-
-After installing Synthesis, the exporter addin should automatically start up when you open Fusion 360 (given that it was selected during the Synthesis install process). Navigate to the Utilities tab and you should see a Synthesis button.
-
-![image_caption](img/fusion/exporter-button.png)
+## Getting Started
+
+### Manual Install
```
Comment (azaleacolburn) [inline tutorials/FusionExporter.md:20]: Requested changes mentioned in installer README

**Review by azaleacolburn — CHANGES_REQUESTED** [review]

**Review by azaleacolburn — APPROVED** [review]
LGTM

**Review by PepperLola — CHANGES_REQUESTED** [review]
Everything looks good, although I think it would be nice to make the manual install instructions [collapsible](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections) since it takes up quite a bit of space. This will be more consequential once there are instructions for the installer.

**Review by PepperLola — APPROVED** [review]
Nevermind we'll come back to it later if necessary

**Pull request merged**